### PR TITLE
Improve README: show intro logo card box

### PR DESCRIPTION
**Pull request by BaseMax**
Hi there,

Just a small PR to improve `README.md` and make this more beautiful.

**Summary:**
Improve README: show intro logo card box

bug-logo

**Problem:**
The logo is too big and ugly and the text is not clear and legible.

Regards,

### CODE REVIEW

**Comment by BaseMax** [comment]
GitHub Dark theme:
dark-theme

GitHub Light theme:
light-theme

**Comment by BaseMax** [comment]
So now the logo has no problem in both themes and looks beautiful. :heart:  (Dark and Light)

https://github.blog/2020-12-08-new-from-universe-2020-dark-mode-github-sponsors-for-companies-and-more/#dark-mode

**Comment by jayschwa** [comment]
FYI, there is a standalone repository for the logo: https://github.com/ziglang/logo

**Comment by BaseMax** [comment]
Thank you Jay.
This does not just include changing the logo, There are other changes in the **readme** file.

We are waiting for the follow-up for the new logo file to be completed. https://github.com/ziglang/logo/pull/12

After that, we continue this PR.

Thanks

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-![ZIG](https://ziglang.org/zig-logo.svg)
-
-A general-purpose programming language and toolchain for maintaining
-**robust**, **optimal**, and **reusable** software.
+<div align="center">
```
Comment (andrewrk) [inline README.md:1]: No html in readme.md please, this is also a plain text document

File: README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+  <a href="https://github.com/ziglang/zig" target="_blank">
+    <img src="https://raw.githubusercontent.com/MaxFork/zig/e7c86677d0754fc884c4242c897a28b7c324f541/logo.svg" alt="ZIG" style="max-width:100%; margin: 0 auto;" width="200">
```
Comment (andrewrk) [inline README.md:3]: this link is to a fork

File: logo.svg [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 140">
```
Comment (andrewrk) [inline logo.svg:1]: I'd like to keep the logo external to this repository, we have a dedicated repository for logo and other images

**Review by andrewrk — CHANGES_REQUESTED** [review]
Thanks for your efforts however I don't think this will work, at least not as is.

**Pull request closed without merging**